src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -578,6 +579,10 @@ private function getErrorMessages(Exception $e)
                     'Admin.Notifications.Error'
                 ),
             ],
+            FreeShippingCartRuleAlreadyExistException::class => $this->trans(
+                'Free shipping cart rule is already added to this cart!',
```

Comment:
@marionf could you check the wording, please ? :cat:

Reply:
```suggestion
                'Free shipping has already been taken into account!',
```